modules/distr-tracing-rn-known-issues.adoc

```diff
@@ -0,0 +1,34 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+* distr-tracing--release-notes.adoc
+////
+
+[id="distr-tracing-rn-known-issues_{context}"]
+= Distributed tracing known issues
+
+////
+Consequence - What user action or situation would make this problem appear (Selecting the Foo option with the Bar version 1.3 plugin enabled results in an error message)? What did the customer experience as a result of the issue? What was the symptom?
+Cause (if it has been identified) - Why did this happen?
+Workaround (If there is one)- What can you do to avoid or negate the effects of this issue in the meantime? Sometimes if there is no workaround it is worthwhile telling readers to contact support for advice. Never promise future fixes.
+Result - If the workaround does not completely address the problem.
+////
+
+The following limitations exist in Red Hat OpenShift distributed tracing platform:
+
+* Apache Spark is not supported.
+
+* Jaeger streaming via AMQ/Kafka is unsupported on IBM Z and IBM Power Systems.
+
+These are the known issues in Red Hat OpenShift distributed tracing platform:
+
+* link:https://issues.redhat.com/browse/TRACING-2057[TRACING-2057] The Kafka API has been updated to `v1beta2` to support the Strimzi Kafka Operator 0.23.0.  However, this API version is not supported by AMQ Streams 1.6.3. If you have the following environment, your Jaeger services will not be upgraded, and you cannot create new Jaeger services or modify existing Jaeger services:
+
+** Jaeger Operator channel: *1.17.x stable* or *1.20.x stable*
+** AMQ Streams Operator channel: *amq-streams-1.6.x*
++
+To resolve this issue, switch the subscription channel for your AMQ Streams Operator to either *amq-streams-1.7.x* or *stable*.
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1918920[BZ-1918920] The Elasticsearch pods does not get restarted automatically after an update. As a workaround, restart the pods manually.
+
+* link:https://issues.redhat.com/browse/TRACING-809[TRACING-809] Jaeger Ingester is incompatible with Kafka 2.3. When there are two or more instances of the Jaeger Ingester and enough traffic it will continuously generate rebalancing messages in the logs. This is due to a regression in Kafka 2.3 that was fixed in Kafka 2.3.1. For more information, see https://github.com/jaegertracing/jaeger/issues/1819[Jaegertracing-1819].
```

Comment:
I had to read these last sentences a couple of times to wrap my head around it. If I am understanding correctly, what do you think about a rewrite like this:

` When there are two or more instances of the Jaeger Ingester with enough traffic, the Jaeger Ingester generates rebalancing messages in the logs. Jaeger Ingester generates these logs because of a regression in Kafka 2.3. This regression was fixed in Kafka 2.3.1.`